app/views/pages/accessibility.html.erb

```diff
@@ -7,65 +7,137 @@
-      <li><%= link_to 'talk to your device', 'https://mcmw.abilitynet.org.uk/talking-to-your-device/', target: :blank, class: 'govuk-link' %></li>
-      <li><%= link_to 'make your device talk to you', 'https://mcmw.abilitynet.org.uk/category/making-your-mobile-or-tablet-talk/', target: :blank, class: 'govuk-link' %></li>
-      <li><%= link_to 'make text larger', 'https://mcmw.abilitynet.org.uk/making-text-larger/', target: :blank, class: 'govuk-link' %></li>
-      <li><%= link_to 'change your colours', 'https://mcmw.abilitynet.org.uk/changing-your-colours/', target: :blank, class: 'govuk-link' %></li>
-      <li><%= link_to 'magnify the screen', 'https://mcmw.abilitynet.org.uk/magnifying-the-screen/', target: :blank, class: 'govuk-link' %></li>
-    </ul>
+    <h2 class="heading-medium">
+      Technical information about this website’s accessibility
+    </h2>
+
+    <p>
+      GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+    </p>
+
+    <h2 class="heading-medium">
+      Compliance status
+    </h2>
+
+    <p>
+      This website is partially compliant with the <%= link_to 'Web Content Accessibility Guidelines version 2.1','https://www.w3.org/TR/WCAG21/', target: :blank, class: 'govuk-link'%> AA standard, due to the non-compliances listed below.
+    </p>
+
+    <h2 class="heading-small">
+      Non-accessible content
+    </h2>
 
+    <p>
+      The content listed below is non-accessible for the following reasons.
+    </p>
+
+    <h3 class="heading-small">
+      Non-compliance with the accessibility regulations
+    </h3>
+
+    <p>
+      Find open data links to several external websites belonging to other government organisations. We cannot guarantee that these websites meet our accessibility requirements. This is not a fail under any specific WCAG criterion, but should be looked at in order to maximise accessibility.
+    </p>
+
+    <p>
+      Some data sets have broken links. This fails WCAG 2.1 success criterion 2.4.4 (Link Purpose).
+    </p>
+
+    <h2 class="heading-medium">
+      How we tested this website
+    </h2>
+
+    <p>
+      We commissioned an expert agency (Digital Accessibility Centre) to review the website.
+    </p>
+
+    <h2 class="heading-medium">
+      What we’re doing to improve accessibility
+    </h2>
+
+    <p>
+      We will adopt improvements to the GOV.UK design system as part of any future work on Find Open Data.
+    </p>
+
+    <h2 class="heading-medium">
+      Preparation of this accessibility statement
+    </h2>
+
+    <p>
+      This statement was prepared on 21 September 2020. It was last reviewed on 22 September 2020.
+    </p>
+
+    <p>
+      This website was last tested in May 2019. The test was carried out by Digital Accessibility Centre
```

Comment:
Missing `.` here.

Reply:
thanks! that's as the example but I think we'll add it in 👍